.travis.yml

```diff
@@ -1,4 +1,4 @@
-dist: xenial
+dist: bionic
```

Comment:
What does this have to do with converting reduce into a library node? ;-)

Reply:
OpenMP implementation doesn't work unless GCC is 6/7 or newer